README.md

```diff
@@ -1,26 +1,14 @@
-# Fx Service Framework
+# :sparkles: Fx [![GoDoc][doc-img]][doc] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov] [![Report Card][report-card-img]][report-card]
```

Comment:
@glibsm looks like its not a github emoji? couldn't find it on: https://gist.github.com/rxaviers/7360908